prow/plugins/trick-or-treat/trick-or-treat_test.go

```diff
@@ -41,12 +41,11 @@ func (c *fakeSnicker) readImage(log *logrus.Entry) (string, error) {
 }
 
 func TestImages(t *testing.T) {
```

Comment:
I'm not sure this "unit test" should exist, it appears to be fetching images from various remote hosts ...?

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/29342/pull-test-infra-unit-test/1649289049657053184

cc @chaodaiG

Reply:
xref: https://github.com/kubernetes/test-infra/pull/19719

Reply:
> > make go-lint is broken on master (bunch of loopclosure issues),
> 
> we run go-lint in `pull-test-infra-verify-lint` which runs `make verify` which is a superset of `make go-lint`, I can't reproduce `make go-lint` failing.

This part also confuses me also, as it does fail in #29305 after #29315 fixed it (the master seemed to be broken, which I find weird as well), and as a bonus it does not fail for you on [Mac but fails on me on Linux](https://github.com/kubernetes/test-infra/pull/29305#issuecomment-1517305684)
Seems flaky to me.

Reply:
If we don't remove the test, we're going to have to update the list of images.

We should do that anyhow, removing the images that no longer exist, but I'm also not so sure about this test or plugin.

Reply:
I removed the failing images, but I agree, fetching ~50 external images on each unit test run is not good.

Reply:
agreed, it was not a well thought out UT, please feel free to remove